web/app/themes/clarity/src/components/c-event-listing/view.php

```diff
@@ -70,12 +71,14 @@
         ?>
 
       <div class="c-event-listing--time">
-        <h2>Time:</h2><?php echo $time; ?>
+        <span>Time:</span>
```

Comment:
I'd maybe  use the `<time>` element here instead of `<span>`? So it would be `<time>Date:</time>` rather then `<span>Date:</span>`.

Reply:
Using `<time>` is a good idea, but it would be the next line `<?php echo $time; ?>` that would have the tag - rather than replacing the `<span>`.